compat/win32/pthread.c

```diff
@@ -57,6 +57,8 @@ pthread_t pthread_self(void)
 	return t;
 }
 
+#ifdef GIT_WIN_XP_SUPPORT
```

Comment:
Perhaps this is a misnomer, but I was under the impression that support for XP was abandoned a while ago.

Restoring support add unnecessary burden on the maintainers of this project. Do you (@rongjiecomputer) plan on being the support person going forward. Be careful before you answer, realize that no other maintainer of contributor is likely to validate that they do not break XP support.